.github/actions/send_slack_notifications/action.yml

```diff
@@ -16,5 +16,5 @@ runs:
         --data '{
         "text": "${{ inputs.MESSAGE }}"
```

Comment:
We need to put this input as an env variable:

env:
  MESSAGE: ${{ inputs.MESSAGE }}

And then substitue like $MESSAGE here.

Reply:
Will update this 🙏